app/(afterLogin)/(okrplanning)/okr/_components/okrDrawer/index.tsx

```
@@ -201,21 +232,66 @@
       />
     </div>
   );
+
+  const keyResultTypes = [
+    { label: 'Milestone', value: 'Milestone' },
+    { label: 'Currency', value: 'Currency' },
+    { label: 'Numerics', value: 'Numeric' },
+    { label: 'Achieved or Not', value: 'Achieved' },
+  ];
+
+  const [addingKeyResult, setAddingKeyResult] = useState(false);
```

ESLint

[error] 'addingKeyResult' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)